common/common.cpp

```diff
@@ -996,11 +1008,20 @@ void gpt_print_usage(int /*argc*/, char ** argv, const gpt_params & params) {
     printf("                          - none: use one GPU only\n");
     printf("                          - layer (default): split layers and KV across GPUs\n");
     printf("                          - row: split rows across GPUs\n");
-    printf("  -ts SPLIT, --tensor-split SPLIT\n");
+    printf("  -ts SPLIT --tensor-split SPLIT\n");
     printf("                        fraction of the model to offload to each GPU, comma-separated list of proportions, e.g. 3,1\n");
-    printf("  -mg i, --main-gpu i   the GPU to use for the model (with split-mode = none),\n");
-    printf("                        or for intermediate results and KV (with split-mode = row) (default: %d)\n", params.main_gpu);
-#endif
```

Comment:
This line might be still necessary as otherwise `LLAMA_SUPPORTS_GPU_OFFLOAD` (line 1001) is not terminated

Reply:
Yes this is fixed in latest code.